test/fixtures/api/swagger.yaml

```diff
@@ -294,6 +294,23 @@ paths:
           schema:
             $ref: "#/definitions/ErrorResponse"
 
+  /sources:
```

Comment:
This will continue being a source of pain, can't we `gitignore` this file and grab it from the git repo if it's missing? It's annoying to always have to replicate API's `swagger.yml` and also easy to forget